apis/sgv2-quarkus-common/src/main/resources/application.yaml

```diff
@@ -62,8 +62,6 @@ quarkus:
       bridge:
         host: localhost
         port: 8091
-        retry: true
-        max-hedged-attempts: 0
```

Comment:
@tatu-at-datastax Can you help here, what do you think should I leave this or remove?

Reply:
Hmmh. I think it is not good to have multiple competing retries, so perhaps leave it as `retry: false` (to document we are not enabling retries at this level).
And then see what performance tests show.